### PR TITLE
[Ft #108]: verify-otp

### PR DESCRIPTION
**Pull request by Elochukwu3**
Made tsc fix on project build

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
+    "build": "tsc",
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": " node dist/app.js",
-    "start:dev": "nodemon ./src/app.ts",
-    "start1": "ts-node -r tsconfig-paths/register src/app.ts"
+    "start:dev": "nodemon ./src/app.ts"
```
Comment (Elochukwu3) [inline package.json:9]: start:dev for starting server using nodemon

File: package.json [hunk 2]
```diff
@@ -3,10 +3,10 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
+    "build": "tsc",
```
Comment (Elochukwu3) [inline package.json:6]: build added for compiling tsc to js on the dist folder

**Review by Elochukwu3 — COMMENTED** [review]
fix bild issue

File: package.json (hunk 2 above)
Comment (slimzyBoom) [reply package.json:6]: Ok nice

File: src/modules/auth/controllers/verifyOtp.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express';
```
Comment (slimzyBoom) [inline src/modules/auth/controllers/verifyOtp.ts:1]: Ok, is it that u added the respective utilities to it's controller file in this place

**Review by slimzyBoom — COMMENTED** [review]
Ok u made changes to the controller file, package.json and routes file

File: src/modules/auth/controllers/verifyOtp.ts (hunk 3 above)
Comment (Elochukwu3) [reply src/modules/auth/controllers/verifyOtp.ts:1]: Having fixed that, just fixed the build error, so as to deploy the it

**Comment by Elochukwu3** [comment]
> Ok u made changes to the controller file, package.json and routes file

**Review by slimzyBoom — APPROVED** [review]

**Pull request merged**